assemble/bin/accumulo-util

```diff
@@ -571,20 +570,17 @@ function load_jars_hdfs() {
   "$HADOOP_PREFIX/bin/hadoop" fs -rm "$SYSTEM_CONTEXT_HDFS_DIR/slf4j*.jar"  > /dev/null
   for f in $(grep -v '^#' "${conf}/tservers")
   do
-    rsync -ra --delete "$ACCUMULO_HOME" "$(dirname "$ACCUMULO_HOME")"
+    rsync -ra --delete "$basedir" "$(dirname "$basedir")"
```

Comment:
This rsync command is currently broken, and has been for some time. Probably should just remove it. I know you're just updating variable names here, but thought I'd mention it, in case you want to clean it up in a follow-on issue.